src/loader/bgw_launcher.c

```diff
@@ -422,6 +447,8 @@ message_start_action(HTAB *db_htab, BgwMessage *message, VirtualTransactionId vx
 			bgw_message_send_ack(message, ACK_FAILURE);
 			return;
 		}
+		else
+			db_he->incremented = true;
```

Comment:
Nit: since there's a `return` in the other branch we don't need the else

Reply:
yep, fixed.